base_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './name_able'
+
+class BaseDecorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
```

Comment:
* Kindly fix this error I get when I try to test your decorator class the methods are not working as expected below is a screenshot of what I got 
![image](https://user-images.githubusercontent.com/52497246/210482642-9ed055da-f0c9-4aa2-8eb4-859a071c98a4.png)